src/rustdoc/calendar.md

```diff
@@ -0,0 +1,5 @@
+# Calendar
+Rustdoc team's calendar is available in the Rust project's
+[`rust-lang/calendar`][calendar_repo] repository.
+
+Currently, the rustdoc team only has regular meetings.
```

Comment:
I don't understand what this sentence is trying to convey: is it implied that it does not have non-regular meeetings? Is it an important info to point out?

Reply:
No indeed, removing it.